doc/nrf/releases/release-notes-latest.rst

```diff
@@ -155,3 +155,56 @@ The following list summarizes the most important changes inherited from upstream
     * ``SYS_CLOCK_HW_CYCLES_TO_NS()``
 
   * Updated :c:func:`k_timer_user_data_get` to take a ``const struct k_timer *timer`` instead of a non-\ ``const`` pointer.
+
+* Drivers:
+
+  * ``DEVICE_INIT()`` macro has been deprecated. Use ``DEVICE_DEFINE()`` instead.
+
+  * ADC:
+
+    * Improved the default routine that provides sampling intervals, to allow intervals shorter than 1 millisecond.
+
+  * Display:
+
+    * Added support for the ILI9488 display.
+    * Refactored the ILI9340 driver to support multiple instances, rotation, and pixel format changing at runtime.
+      Configuration of the driver instances is now done in Devicetree.
+    * Enhanced the SSD1306 driver to support communication via both SPI and I2C.
+
+  * Flash:
+
+    * Modified the nRF QSPI NOR driver so that it supports also nRF53 Series SoCs.
```

Comment:
This one is already added in https://github.com/nrfconnect/sdk-nrf/pull/3465 - can you coordinate with @nvlsianpu ?

Reply:
I guess it should be removed from there and stay here. Is that okay with you @nvlsianpu?